common/app/templates/inlineJS/blocking/applyRenderConditions.scala.js

```diff
@@ -64,6 +64,45 @@
         }).join(' ');
     }
 
+    /*
+        This is a shortened version of shouldSeeReaderRevenue() from
+        user-features.js. Since we are blocking rendering at this time we
+        can't/ don't want to online all required JS from this module.
+    */
+    function hideReaderRevenue() {
+        var getCookieValue = function(name) {
+            var val = document.cookie.match('(^|;)\\s*' + name + '\\s*=\\s*([^;]+)');
+            return val ? val.pop() : undefined;
+        };
+
+        var contribution = {
```

Comment:
True that - talked to @sndrs already. I'll do a bit more refactoring, to minimize the footprint of this.

👍